web-app/cypress/support/commands.ts

```diff
@@ -36,3 +36,35 @@
 //     }
 //   }
 // }
+
+Cypress.Commands.add('injectAuthenticatedUser', () => {
+  cy.window()
+    .its('store')
+    .invoke('dispatch', {
+      type: 'userProfile/loginSuccess',
+      payload: {
+        fullName: 'Valery',
+        email: 'testuser@gmail.com',
+        isRegistered: true,
+        isEmailVerified: true,
+        organization: '',
+        isRegisteredToReceiveAPIAnnouncements: false,
+        isAnonymous: false,
+        refreshToken:
+          'AMf-vBwvDFwWA77IKuTUdQ9eZ7sCalLb3LPfHupyvvI91SYcImb_e5R417gPIZbVJxaJUSvMDqHWlQaMMZQZkkahT9zFW3FXymDMmvSzKB-NKO2X_lw1yCP_YyegslW6Wl4y2PRG_gEyUHxpESrjDVI_scIDIvfHaoH97-GraKJFgmCo61QzmYyMY8vyaOxyH9ovcVwdiy-KLpRbU-B8VrtrRwKg-o7BWggTlIRXu5hgtG34lS8GCdM',
```

Comment:
Is this a _valid_ dev refresh token? If it is, we must have it as a variable injected by the ci instead of plain text.